cloudify/event.py

```diff
@@ -75,7 +75,12 @@ def operation_info(self):
 
     @property
     def text(self):
-        message = self._event['message']['text'].encode('utf-8')
+        message = self._event['message']['text']
+        try:
+            message = message.encode('utf-8')
```

Comment:
encoding here is still wrong

Reply:
@tehasdf as discussed, while wrong, this should be resolved by a different JIRA (or in the Py3 branch).